R/error-checks.R

```diff
@@ -414,14 +414,28 @@ validateSameOutputsBetweenSets <- function(simulationSets, logFolder = NULL) {
     pkParametersTable <- getPKParametersInSimulationSet(set)
     # In case output or pkParameters are in different orders
     pkParametersTable <- pkParametersTable[order(pkParametersTable$path, pkParametersTable$pkParameter), ]
-    if (is.null(pkParametersTableRef) ||
-      all(
-        pkParametersTable$path == pkParametersTableRef$path,
-        pkParametersTable$pkParameter == pkParametersTableRef$pkParameter
-      )) {
+
+    if (is.null(pkParametersTableRef)) {
       pkParametersTableRef <- pkParametersTable
       next
     }
+
+    if(all(pkParametersTable$path == pkParametersTableRef$path)){
+      pkParametersTableTest <- NULL
+      for (n in seq_along(pkParametersTable$pkParameter)){

```

Comment:
Can you replace `n` by `pkParameterIndex` ?

Reply:
Done